vuepress/docs/next/tutorials/consume/multitenancy.md

```diff
@@ -52,26 +51,26 @@ sed -i'' 's/\/\/YOUR-APP-NAME\./\/\/YOUR-TENANT-ID\.YOUR-APP-NAME\./g' keycloak-
 
 5. Log in to your [Keycloak admin console](../../docs/consume/identity-management.md#authorization), typically located at `http(s)://YOUR-APPNAME.YOUR-HOST-NAME/auth`.
 
-6. Go to `Select realm` in the top left nav → `Add Realm` → select your `keycloak-realm.json` file. Click `Create`.
+6. Find `Entando` in the left sidebar, below the logo.  Click `Add Realm` and select your `keycloak-realm.json` file. Click `Create`.
 
 7. In the new realm, go to `Clients` → Click the Client ID with `YOUR-APP_NAME`. Under the `Credentials` tab, regenerate the `Secret`. 
 
-> Note: The Secret for this client will be needed in the `entando-tenants-config` Secret below for `YOUR-TENANT-KC-SECRET`.
+> Note: The Secret for this client will be needed in the `entando-tenants-secret.yaml` for `YOUR-TENANT-KC-SECRET` below.
 
-8. Regenerate the Secret for Client ID `YOUR-APP-NAME-de` as well. 
+8. Regenerate the Secret for Client ID `YOUR-APP-NAME-de`. 
 
-9. Go to `Manage` → `Users`. Click `View all users` and choose the `admin` user. [Grant the user Role Mapping](../../docs/consume/identity-management.md#authorization) with these assignments: Client `realm-management` and Role `manage-realm`.
+9. Go to `Manage` → `Users`. Click `View all users` and choose the `admin` user. Go to [Role Mapping](../../docs/consume/identity-management.md#authorization) and make this assignment: Client `realm-management` and Role `manage-realm`.
 
 10. (Optional) Go to `Credentials` and reset the password for the `admin` user.
 
 ### Content Delivery Server (CDS)
-Each tenant requires its own set of CDS resources. Follow the [CDS tutorial](./cds.md) to prepare these resources.
+Each tenant requires its own set of CDS resources. Follow the [CDS tutorial](./cds.md) to prepare these resources for each secondary tenant.
 
 ### Solr
-Each tenant requires a dedicated Solr core or collection. Follow the [Solr tutorial](./solr.md#generate-the-core) to setup the core for the tenant. 
+Each tenant requires a dedicated Solr core or collection. Follow the [Solr tutorial](./solr.md#generate-the-core) to generate the core and schema for each secondary tenant. 
 
 ### Database
-Each tenant requires a new database schema for the Entando tables related to the page structure, web content, widgets, etc. The following steps provide a way to export the schemas in a default Entando installation using the Entando-generated PostgreSQL instance. The exact commands will differ for external databases or different DBMS types.
+Each tenant requires a new database schema for the Entando tables related to the page structure, web content, widgets, etc. The following steps provide a way to export the schemas in a default installation using the Entando-generated PostgreSQL instance. The exact commands will differ for external databases and other DBMS.
 
 1. Determine the name of your PostgreSQL pod (YOUR-POSTGRESQL-POD):
 ``` bash
```

Comment:
Within point n. 2 we probably need to add something about the CM database. I think we could extend the query by adding this suffix before the semi colon: `OR schema_name LIKE '%dedb%'`
Then we should add a line to resulting example values, for example `quickstart_dedb_9472` mentioning it in the following lines as `YOUR-SCHEMA-3` and changing the string `-n YOUR-SCHEMA-2` in the instruction at point 3 to `-n YOUR-SCHEMA-2 -n YOUR-SCHEMA-3`
Then at point 4 `sed -i'' 's/YOUR-SCHEMA-1/YOUR-TENANT-ID/g; s/YOUR-SCHEMA-2/YOUR-TENANT-ID/g' db_export.sql` should become `sed -i'' 's/YOUR-SCHEMA-1/YOUR-TENANT-ID/g; s/YOUR-SCHEMA-2/YOUR-TENANT-ID/g; s/YOUR-SCHEMA-3/YOUR-TENANT-ID/g' db_export.sql`